### ISSUE TITLE
Keeps crashing during Kill Spree

### ISSUE THREAD

**Issue by SemperF1** [fragtrane/Payday-2-Mod-Updates#1]
It ist crashing during this Kill Spree.
I only use Wolf HUD and Item Hunter

**Comment by fragtrane** [comment]
I think you mean Crime Spree right? If you can post your crash log that would be helpful, you can find it here:
```
%LocalAppData%\PAYDAY 2\crash.txt
```

Can you give any more information, e.g. what level you were playing and what you were doing when the crash happened?

Also check that you are using the latest version of Item Hunter (v1.2).

**Comment by SemperF1** [comment]
Yes i mean Crime Spree. No sorry i Removed Item Hunter cuz i want to play without Craches while Crime Spree cuz you have to Pay the Coins for Failing. But i Played aftershock, when i pressed the Hotkey it closes my game. It Happend 3 Times in a row. I try to recreate the error today.

**Comment by SemperF1** [comment]
Application has crashed: access violation

-------------------------------

Callstack:

         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     zip_get_name                                        
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     zip_get_name                                        
         payday2_win32_release  (???)     ???                                                 


-------------------------------

Current thread: Main
Script stack:
               load_packages()  lib/setups/gamesetup.lua:372                        
                                core/lib/setups/coresetup.lua:323                   

-------------------------------

System information:
	Application version : 1.94.862
	CPU : Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz (3 cores); SSE; SSE2; SSE3; SSSE3; SSE4.1; SSE4.2
	DirectX : 12.0 
	GPU : NVIDIA GeForce GTX 1070 / nvldumd.dll[26.21.14.4575]
	Language : german
	Memory :     16326MB 264KB
	OS : 6.2.9200 () 0x100-0x1 (64 bits)
	Physics : threaded
	Renderer : DX9 threaded
	Sound : ASUSTeK (Lautsprecher (ASUS Essence STX II Audio Device))

**Comment by SemperF1** [comment]
I Hope i will help you.

**Comment by SemperF1** [comment]
Here is another Crash report

Application has crashed: access violation

-------------------------------

Callstack:

         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 


-------------------------------

Current thread: Main
Script stack:
              check_keep_wps()  @mods/Item                                          
                                Hunter/lua/setup.lua:310                            
                    clean_wp()                                                      
                    @mods/Item  Hunter/lua/setup.lua:238                            
                        func()  @mods/Item                                          
                                Hunter/lua/objectinteractionmanager.lua:15          
                 remove_unit()                                                      
@mods/base/req/core/Hooks.lua:267  set_active()                                        
                                lib/units/interactions/interactionext.lua:542       
                     destroy()                                                      
lib/units/interactions/interactionext.lua:652  destroy()                                           
                                lib/units/interactions/interactionext.lua:796       
lib/units/interactions/interactionext.lua:1231

**Comment by fragtrane** [comment]
Thanks for the information. I have tried playing Crime Spree on Aftershock as both client and host several times, but I could not reproduce the crash.

The first crash log doesn't contain anything pointing to Item Hunter so I am not sure what is going on.

I have seen the second crash though. Can you check if you are using Item Hunter v1.2? Because I tried to fix this problem in v1.2, but if it is still not fixed I will have to take another look at it.
